pydoover/cloud/reports/application.py

```diff
@@ -0,0 +1,258 @@
+import asyncio
+from datetime import datetime, timedelta
+from zoneinfo import ZoneInfo
+import logging
+import time
+import croniter
+
+from ..processor.types import MessageCreateEvent, DeploymentEvent, ScheduleEvent
+from ..processor.application import Application as ApplicationBase
+from ...config import Schema
+
+
+log = logging.getLogger()
+
+SPLIT_MESSAGES_LIMIT = 100  # 1 hour
+
+
+class Application(ApplicationBase):
+    def __init__(self, config: Schema | None):
+        super().__init__(config)
+
+        self.period_start = None
+        self.period_end = None
+        self.devices = None
+        
+    async def on_message_create(self, event: MessageCreateEvent):
+        log.error("Report Genertator does not support message create events")
+        pass
```

Comment:
I'd probably either suggest raising a `RuntimeError` to make it really clear to the user, or if it doesn't actually matter then just omit the function